File: PYTHON/PRAK604-2210817220029-SitiAinurRahmawati.py
kunci=list(input())
pesan=list(input())
salah=0
benar=0
if len(kunci) != len(pesan) :
    print("Panjang kalimat berbeda, pesan palsu")
else :
    for i in range (len(kunci)) :
        if kunci[i] == pesan[i] :
            print("*", end = "")
            benar=benar+1
        else :
            print("#", end = "")
            salah=salah+1
    print()
    print("* = ", benar)
    print("# = ", salah)
    if benar >= salah :
        print("Pesan Asli")
    elif benar < salah :
        print("Pesan Palsu")

File: C/PRAK604-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>
#include <string.h>
int main () {
    char pw[99], ch[99];
    scanf("%[^\n]%*c", &pw);
    scanf(" %[^\n]%*c", &ch);
    int len1, len2, i, a=0, b=0;
    len1 = strlen(pw);
    len2 = strlen(ch);
    if (len1 != len2) {
        printf("Panjang kalimat berbeda, pesan palsu");
    } else { 
        for (i=0;i<len1;i++) {
            if (pw[i]==ch[i]) {
                printf("*");
                a++;
            } else {
                printf("#");
                b++;}}
        printf("\n* = %d", a);
        printf("\n# = %d", b);
    if (a>=b) {
        printf("\nPesan Asli");}
    else if (a<b) {
        printf("\nPesan Palsu");} }
return 0;}